### ISSUE TITLE
Parts of an image could end up being redacted.

### ISSUE THREAD

**Issue by fabioz** [robocorp/robocorp#399]
### Target components

log

### What is wrong

If an html is being added (such as an image), some parts can end up being redacted (which may not be what's wanted).

### System info

Any

**Comment by fabioz** [comment]
This is actually an important issue (that we should've tackled already).

**Comment by Maxhodges** [comment]
we're paying for RoboCloud and it was a costly migration from Robot Framework to Python, and now over a year later screenshots still not working... can we get a discount? We're not getting the same service as when we signed up!

**Comment by Maxhodges** [comment]
> This is actually an important issue (that we should've tackled already).

Yes, it's important. Will someone give it the attention it deserves? Or is RoboCLoud development dead and we should migrate away?

**Comment by fabioz** [comment]
Thank you for the heads up, this was really missing.

It's now fixed in `robocorp-log 3.0.0` (if you use the `robocorp` bundle, it's also in `robocorp 3.0.0`).
